game/combo.go

```diff
@@ -64,6 +65,7 @@ type Player interface {
 
 type Game interface {
 	Play() Player
+	Board() Board
```

Comment:
I like the GameResult idea.  I initially did it with a second return value but worried about interface bloat once you add a game clock.  I can make the change.